src/content/docs/supporters/get-started.mdx

```diff
@@ -0,0 +1,223 @@
+</Steps>
+
+Contact Interledger Wallet at support@interledger.app if you need additional help.
+
+## Extension states
+
+When the **Continuous payment** toggle is on, the extension icon appears in color. When the toggle is off, the icon appears in gray. Whether a page is web monetized does not affect the color of the icon.
+
+#### Continuous payment toggled on
+
+| Icon | State |
+| :--: | ----- | 
+| <div style="padding-left: 5px;">![Continuous payments turned on in extension, extension is paying](/img/docs/extension/enabled-monetized.png 'Active icon state with green circle and checkmark')</div> | Page is web monetized and the extension is continuously paying                                   |
+| <div style="padding-left: 5px;">![Continuous payments turned on in extension, extension is not paying and needs user attention](/img/docs/extension/enabled-attention.png 'Active icon state with orange circle and exclamation mark')</div> | Page is web monetized, but the extension needs user attention and is **not** continuously paying |
+| <div style="padding-left: 5px;">![Continuous payments turned on in extension, page is not web monetized](/img/docs/extension/enabled-unmonetized.png 'Active icon state with red circle and x')</div> | Page is **not** web monetized, the extension is **not** continuously paying |
+
+#### Continuous payment toggled off
+
+|                                                                                                         Icon                                                                                                         | State                                                                                            |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------ |
+|                <div style="padding-left: 5px;">![Continuous payments turned off in extension](/img/docs/extension/disabled-monetized.png 'Inactive icon state with green circle and checkmark')</div>                | Page is web monetized, but the extension is **not** continuously paying                          |
+| <div style="padding-left: 5px;">![Continuous payments turned off in extension, needs user attention](/img/docs/extension/disabled-attention.png 'Inactive icon state with orange circle and exclamation mark')</div> | Page is web monetized, but the extension needs user attention and is **not** continuously paying |
+|      <div style="padding-left: 5px;">![Continuous payments turned off in extension, page is not web monetized](/img/docs/extension/disabled-unmonetized.png 'Inactive icon state with red circle and x')</div>       | Page is **not** web monetized, so the extension is **not** continuously paying                   |
+
+## FAQ
```

Comment:
Would be good to add another entry for:
1. Why the balance in extension is not updating on payments? Known issue [#737](https://github.com/interledger/web-monetization-extension/issues/737).
2. The balance shown in your wallet (your wallet's balance) is unrelated to the balance shown in extension (the grant balance). The grant balance can be more than your wallet balance even.